### PR TITLE
Feature/3 user management service

### PR DESCRIPTION
**Pull request by SwetShah**
#3-milestone1: UserManagementService code with login and register feature.

### CODE REVIEW

File: UserManagement/.project [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (vinayakasg18) [inline UserManagement/.project:1]: Kindly add all folders and files which are not required for the project to .gitingnore file.

File: UserManagement/.settings/org.eclipse.jdt.core.prefs [hunk 2]
```diff
@@ -0,0 +1,8 @@
+eclipse.preferences.version=1
```
Comment (vinayakasg18) [inline UserManagement/.settings/org.eclipse.jdt.core.prefs:1]: Same here.

File: UserManagement/hs_err_pid76100.log [hunk 3]
```diff
@@ -0,0 +1,44 @@
+#
```
Comment (vinayakasg18) [inline UserManagement/hs_err_pid76100.log:1]: Remove log files from commit.

File: UserManagement/src/main/java/com/UserManagementBootstrapper.java [hunk 4]
```diff
@@ -0,0 +1,21 @@
+package com;
```
Comment (vinayakasg18) [inline UserManagement/src/main/java/com/UserManagementBootstrapper.java:1]: Kindly add logging into the next milestone.

File: UserManagement/src/main/resources/application.properties [hunk 5]
```diff
@@ -0,0 +1,4 @@
+#Local MongoDB config
```
Comment (vinayakasg18) [inline UserManagement/src/main/resources/application.properties:1]: Create a technical story to externalize the application .properties in the next milestone.

File: UserManagement/src/main/resources/application.properties [hunk 6]
```diff
@@ -0,0 +1,4 @@
...
+spring.data.mongodb.port=27017
+spring.data.mongodb.host=localhost
+spring.data.mongodb.database=admin
```
Comment (vinayakasg18) [inline UserManagement/src/main/resources/application.properties:4]: Kindly remove the .jar file from the commit.

**Review by vinayakasg18 — CHANGES_REQUESTED** [review]

File: UserManagement/.project (hunk 1 above)
Comment (SwetShah) [reply UserManagement/.project:1]: Will do it. Thanks

File: UserManagement/hs_err_pid76100.log (hunk 3 above)
Comment (SwetShah) [reply UserManagement/hs_err_pid76100.log:1]: Yes, Sure!

File: UserManagement/src/main/java/com/UserManagementBootstrapper.java (hunk 4 above)
Comment (SwetShah) [reply UserManagement/src/main/java/com/UserManagementBootstrapper.java:1]: Thanks for pointing this out. 
will do it

File: UserManagement/src/main/resources/application.properties (hunk 5 above)
Comment (SwetShah) [reply UserManagement/src/main/resources/application.properties:1]: Will look for different options for externalizing and move ahead accordingly

File: UserManagement/src/main/resources/application.properties (hunk 6 above)
Comment (SwetShah) [reply UserManagement/src/main/resources/application.properties:4]: Got it!

**Review by vinayakasg18 — APPROVED** [review]

**Review by vinayakasg18 — APPROVED** [review]
Approved the changes.

**Pull request merged**